src/Controller/Component/SearchComponent.php

```diff
@@ -99,8 +99,13 @@ public function beforeRender()
 
         $controller = $this->getController();
         try {
+            // check to ensure modelclass is set before loading it
```

Comment:
This comment isn't necessary, the code block is simple enough to be understood by itself.